src/lib/image/bootstrap/busybox/busybox.c

```diff
@@ -0,0 +1,45 @@
+/* 
```

Comment:
If we port them to C, they would almost entirely be made up of shell commands, so I would argue there isn't much of a need to. Do you concur?